.fixtures.yml

```diff
@@ -5,7 +5,7 @@ fixtures:
       ref: 5.0.1
     stdlib:
       repo: git://github.com/puppetlabs/puppetlabs-stdlib.git
-      ref: 4.25.1
```

Comment:
`metadata.json` would also need to be updated.

Reply:
It was. Now has `>= 5.1.0 < 7.0.0`